eplaunch/interface/frame_main.py

```diff
@@ -490,7 +490,7 @@ def _handle_focus_in(self, _event) -> None:
         # the stale attribute, which should be accomplished with just the call here.
         # It's possible we also need to check the _event.widget to make sure we are only
         # calling this when the main window is focused.
-        self._update_file_list()
+        pass  # self._update_file_list()
```

Comment:
From testing, it looks like we shouldn't need that call.  May be worth another review and/or cleanup soon.

Reply:
This surprises me a bit but if it works without that is ok.

Reply:
OK, I think I know what's happening.  When we are actually running workflows, or navigating directories, or whatever, we intentionally call this _update_file_list function.  The only reason we would need to update the file list during focus is if they actually change the files externally in the background and expect EP-Launch to just constantly update the file list.  I personally think this is too far.  If they are using EP-Launch to navigate and run things, then they will see the updates and everything will be fine.  If they do stuff using other programs, worst case is that they need to refresh EP-Launch when they come back to the tool.

If I leave this in, then we end up with lots and lots of excess calls to update the file list, which is at best a waste of time, and at worst, flickers or weird behavior.